kbase-extension/static/kbase/js/widgets/narrative_core/upload/stagingAreaViewer.js

```diff
@@ -19,7 +19,7 @@ define([
     'uuid',
     'jquery-dataTables',
     'select2'
-], function (
+  ], function (
```

Comment:
For some reason all of these whitespace changes got picked up by git, apologies for the noise